src/Microsoft.Health.Abstractions/Microsoft.Health.Abstractions.csproj

```diff
@@ -5,6 +5,11 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.3.0" />
```

Comment:
I don't think EventGrid should be added to Abstractions...

Reply:
Originally it wasn't dependent but since we didn't want to have IEvent based on earlier comment here by Teng, I had to get rid of it and that introduces the dependency on EventGrid. Let me know.

Reply:
Let me double check.

Reply:
This will introduce EventGrid as a dependency on all downstream projects that depend on Abstractions, I don't think this would be idea. Is there going to be a `Microsoft.Health.EventGrid` common library also?

Reply:
Okay I remember now. Originally there was no dependency because I had abstracted EventGridEvent with IEvent. But since Teng wanted to get rid of that in earlier comments above, it introduced the dependency for IEventGridPublisher under Microsoft.Health.Abstractions.Features.Events because instead of taking IEvent its now taking EventGridEvent.

Reply:
One fix would be to move the IEventGridPublisher to  Microsoft.Health.Core.Features.Events

Reply:
Thanks for catching this @brendankowitz . I have moved it out of abstractions project, given we already got rid of IEvent. Fixed.